app/controllers/api/v1/netflix_shows_controller.rb

```diff
@@ -1,9 +1,9 @@
 class Api::V1::NetflixShowsController < ApiController
   def index
-    render json: NetflixShow.all
+    render json: NetflixShow.all, each_serializer: NetflixSerializer
```

Comment:
Not a requested change, but this should be able to find the `NetflixSerializer` by default.  `each_serializer` will be most useful if we need to point it at a serializer that it won't go to by default

Reply:
Removed each_serializer: NetflixSerializer